src/main/java/io/openbpm/control/view/processdefinition/DeleteProcessDefinitionView.java

```diff
@@ -56,6 +59,9 @@ public void onInit(final InitEvent event) {
     @Subscribe
     protected void onBeforeShow(BeforeShowEvent event) {
         deleteProcessInstancesCheckBox.setValue(true);
+
+        long countByProcessDefinitionId = processInstanceService.getCountByProcessDefinitionId(processDefinitionId);
+        deleteProcessInstancesCheckBox.setEnabled(countByProcessDefinitionId == 0);
```

Comment:
Add a similar mechanism for bulk operations:
![image](https://github.com/user-attachments/assets/6cf3a148-76a4-4b95-b079-eebaf139c0a3)